Portal/Controllers/WasteController.cs

```diff
@@ -138,5 +139,27 @@ public async Task<IActionResult> Tonnes(ExportTonnageViewModel exportTonnageView
 
             return RedirectToAction("Index", "Home");
         }
+
+        [HttpGet]
+        public async Task<IActionResult> BaledWithWire(int? id)
+        {
+            if (id == null)
+                return NotFound();
+
+            var model = await _wasteService.GetBaledWithWireModel(id.Value);
+            return View(model);
+        }
+        
+        [HttpPost]
+        public async Task<IActionResult> BaledWithWire(BaledWithWireModel baledWithWire)
+        {
```

Comment:
Method parameter name 'baledWithWire' is the same as the action name. This may result in incorrect model binding. Consider renaming the parameter or the property to avoid conflicts. There is a warining as well. renaming this should resolve it.

Reply:
Done.